src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/CommandLineParser.cs

```diff
@@ -0,0 +1,248 @@
+//-----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//-----------------------------------------------------------------------------
```

Comment:
Can we please update the header of all .cs files with the MIT license header? Did we run code formatting already? It's okay to do that in a separate PR. I just call it out.

Reply:
I'll run CodeFormatter in a separate PR.